src/applications/vaos/express-care/redux/selectors.js

```diff
@@ -0,0 +1,20 @@
+export function selectExpressCareNewRequest(state) {
+  return state.expressCare.newRequest;
+}
+
+export function selectExpressCareFormData(state) {
+  return selectExpressCareNewRequest(state).data;
+}
+
+export function getExpressCareFormPageInfo(state, pageKey) {
+  const newRequest = selectExpressCareNewRequest(state);
+  return {
+    schema: newRequest.pages[pageKey],
+    data: newRequest.data,
+    pageChangeInProgress: newRequest.pageChangeInProgress,
+  };
+}
+
+export function selectExpressCare(state) {
```

Comment:
This is new, and only used on the details page, where the submission status is needed.

Reply:
Totally a nit, but could maybe move this top the top and have `selectExpressCareNewRequest` use it as well.